capstone/fitness/src/main/java/com/google/sps/servlets/CreateProfileServlet.java

```diff
@@ -0,0 +1,97 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query.FilterOperator;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import java.io.IOException;
+import javax.servlet.RequestDispatcher;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/create-profile")
+public class CreateProfileServlet extends HttpServlet {
+  
+  // Constants used for datastore.
+  final static String USER_ENTITY = "user";
+  final static String PROGRESS_PROPERTY = "sessions";
+  final static String NAME_PROPERTY = "name";
+  final static String AGE_PROPERTY = "age";
+  final static String MARATHON_LENGTH_PROPERTY = "marathonLength";
+  final static String WEEKS_PROPERTY = "weeksToPrepare";
+  final static String INITIAL_TIME_PROPERTY = "initialTime";
+  final static String GOAL_TIME_PROPERTY = "goalTime";
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException, ServletException {
+
+    // We get the user email everytime we do a request because if we had it in.
```

Comment:
Please clarify this comment.

Reply:
That comment addressed something that I had removed, and I think I forgot to remove the top line. Removed it now.